Rollbar/NetStandard/RollbarConfigurationLoader.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace Rollbar.NetStandard
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Reflection;
+
+    using Rollbar.Common;
+    using Rollbar.Telemetry;
+
+    /// <summary>
+    /// Class RollbarConfigurationLoader.
+    /// Implements the <see cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    /// </summary>
+    /// <seealso cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    public class RollbarConfigurationLoader 
+        : IRollbarConfigurationLoader
+    {
+        /// <summary>
+        /// The trace source
+        /// </summary>
+        private static readonly TraceSource traceSource = new TraceSource(typeof(RollbarConfigurationLoader).FullName);
+
+        /// <summary>
+        /// The known assembly names
+        /// </summary>
+        private readonly static string[] knownAssemblyNames = {
+            "Rollbar.App.Config",
+            "Rollbar.AppSettings.Json",
+            };
+
+        /// <summary>
+        /// The known loader type name
+        /// </summary>
+        private readonly static string knownLoaderTypeName = nameof(RollbarConfigurationLoader);
+
+        /// <summary>
+        /// The configuration loader type
+        /// </summary>
+        private readonly static Type configurationLoaderType = null;
+
+        /// <summary>
+        /// The loader
+        /// </summary>
+        private readonly IRollbarConfigurationLoader _loader = null;
+
+        /// <summary>
+        /// Initializes static members of the <see cref="RollbarConfigurationLoader" /> class.
+        /// </summary>
+        static RollbarConfigurationLoader()
+        {
+            string sdkAssembliesPath = RuntimeEnvironmentUtility.GetSdkRuntimeLocationPath();
+            traceSource.TraceInformation($"Rollbar SDK runtime location: {sdkAssembliesPath}");
+            Assembly assembly = null;
+            foreach(var name in RollbarConfigurationLoader.knownAssemblyNames)
+            {
+                string dllName = $"{name}.dll";
+                string dllFullPath = Path.Combine(sdkAssembliesPath, dllName);
+                if (!File.Exists(dllFullPath))
+                {
+                    continue;
+                }
+
+                traceSource.TraceInformation($"Located optional Rollbar SDK assembly: {dllFullPath}");
+                assembly = Assembly.LoadFrom(dllFullPath);
+                if (assembly != null)
+                {
+                    traceSource.TraceInformation($"Loaded optional Rollbar SDK assembly: {dllFullPath}");
```

Comment:
Nit: to the end of this message, I'd add "Trying to use" so that's clear what should be next.

Reply:
same as the one above...